src/components/homepage/HomepageList.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useNavigate } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { getCompanies } from '../../redux/companySlice';
+import HomepageItem from './HomepageItem';
+import './HomepageList.css';
+
+const HomepageList = () => {
+  const navigate = useNavigate();
+  const [search, setSearch] = useState('');
+  const { companyList, status, error } = useSelector(
```

Comment:
- [ ] Great job adding the test cases. At the moment, you have 1 test case failing. Kindly fix it.
![image](https://github.com/aster-alemu/metrics-web-app/assets/62849872/dbb859b2-3a3b-4600-9d0a-307821017fee)